blog/2023/12-10-android-tv-16/index.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Android TV version 0.16
+authors:
+  - nielsvanvelzen
+slug: androidtv-v0.16.0
+tags: [release, android-tv]
+---
+
+It has been a busy year with a lot of work on the Android TV app. After an extended beta period today is finally the day to share this work
+to the public. Read along to learn everything about improved music playback, the new screensaver and more!
+
+\- Niels
+
+{/* truncate */}
+
+## Significant changes
+
+In this post I will focus on a few of the popular and significant changes. As always a lot more changes were made both visible and
+invisible; check out the full changelog to discover the full extent of this release.
+
+### Music playback
+
+Playing media is the key feature of Jellyfin. Up until now the Android TV app had many issues playing every possible format. The code
+was also hard to work with making it challenging to fix the issues or implement new features. To encounter this we decided to rewrite all
+playback code completely from scratch. In this release the initial results are ready for daily use and playing music will use the new
```

Comment:
>  rewrite all playback code completely from

or

>  rewrite all music related playback code completely from

Reply:
No, everything. All old code will be gone. With 0.16 we just release it for music and 0.17 will use it for everything, completely removing the old junk.

Reply:
Then i would make it a bit more clear, as this paragraph is below the music header.

Reply:
I think it's already clear enough, the paragraph starts talking about all playback code for some backstory and ends with "In this release the initial results are ready for daily use and playing music will use the new playback code!".